pkg/test/test.go

```diff
@@ -90,15 +102,23 @@ func (m *Mocks) runComponent(component mockComponent) {
 	m.waitGroup.Add(1)
 	go func() {
 		defer m.waitGroup.Done()
-		component.Start()
+
+		err := component.Start()
+		if err != nil {
+			m.logger.Fatalf(err, "failed to start component")
```

Comment:
This is going to call `os.Exit(1)` which will completely skip the deferred shutdown of the containers.

Reply:
you're right, but since the components by default do expire after 1 minute and due to the fact that we do use dynamic port allocation i didn't really see an issue here. But i am fine to use Panicf instead